docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-2.md

```diff
@@ -17,8 +17,7 @@ management workspace (WMW).  Start by creating a WMW for the common
 workload, with the following commands.
 
 ```shell
-kubectl ws root
-kubectl kubestellar ensure wmw wmw-c
+IN_CLUSTER=false SPACE_MANAGER_KUBECONFIG=~/.kube/config kubectl kubestellar ensure wmw wmw-c
```

Comment:
It looks like some updates are needed to https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/commands/#creating-a-workload-management-workspace

Reply:
We will handle this as part of PR3 which continues the work on example1.